src/components/Engines/Engines.tsx

```diff
@@ -13,7 +13,7 @@ import node from './img/node.png';
 const Engine: React.FC = () => {
   const { packageMeta } = useContext(DetailContext);
 
-  const engines = packageMeta && packageMeta.latest && packageMeta.latest.engines;
+  const engines = packageMeta?.latest?.engines;
 
   if (!engines || (!engines.node && !engines.npm)) {
```

Comment:
```suggestion
  if (!engines?.node && !engines?.npm) {
```
I know it's only a PoC but since it's right below, this `if` can also be simplified with optional chaining